src/components/beat.js

```diff
@@ -47,9 +47,9 @@ AFRAME.registerComponent('beat-system', {
 
   // Will be overridden.
   verticalPositions: {
-    bottom: 0.95,
-    middle: 1.35,
-    top: 1.75
+    bottom: 0.90,
```

Comment:
these values don't actually do anything. in my current branch, i stubbed out this object to reduce future confusion

Reply:
Moved new values to where positions are calculated